tools/clang/test/CodeGenDXIL/hlsl/linalg/.clang-format

```diff
@@ -0,0 +1,2 @@
+BasedOnStyle: LLVM
```

Comment:
I think we usually turn off clang-format for tests?  I had it enabled while I was working on the proposal so examples would be formatted nicely for the document, but it might be preferable to just not use it on these test files so you can avoid having to turn it off/on again around CHECK lines.

I don't mind what you do here, but just wanted to be sure you weren't suffering with a choice I made now that the requirements have changed.

Reply:
Yep just a copy paste, unsure if it was needed or not. I'll remove clang-format.